packages/promise-kit/test/test-promise-kit.js

```diff
@@ -58,7 +60,7 @@ function testRacePromise(t, candidate) {
     const result = Promise.resolve(candidate);
 
     bothResults = Promise.allSettled([raced, result]);
-    fr.register(raced, collected.resolve);
+    fr.register(raced, () => collected.resolve(undefined));
```

Comment:
Nothing to do with this really, but it's one of my biggest nit with tsc that `() => void` is incompatible with `() => any`. If it wasn't a test, I'd have asked for correcting the type of `collected` to `PromiseKit<void>` instead as I dislike changing runtime for types.